src/test/e2e/api/admin/api-token.auth.e2e.test.ts

```diff
@@ -190,6 +204,35 @@ test('Token-admin should be allowed to create token', async () => {
     await destroy();
 });
 
+test('An admin token should be allowed to create a token', async () => {
```

Comment:
@chriswk adding this additional case, which is different from the setup of the other tests, to use the authorization header instead of the preHook that injects a user (not a token). Let me know what you think about this, maybe the test should be in a different test-suite